app/models/Arrival.scala

```diff
@@ -27,12 +30,30 @@ case class Arrival(
   status: ArrivalStatus,
   created: LocalDateTime,
   updated: LocalDateTime,
-  messages: Seq[MovementMessage],
+  messages: NonEmptyList[MovementMessage],
   nextMessageCorrelationId: Int
 )
 
 object Arrival {
 
+  @deprecated("use NonEmptyList[MovementMessage] over Seq[MovementMessage]", "now")
+  def apply2(arrivalId: ArrivalId,
```

Comment:
How come we're marking this as deprecated and not just changing it to use a NonEmptyList?

Reply:
Oops this was while I was trying to get back to a compiling state, I'll get rid of that

Reply:
Removed